lua/damagelogs/client/rdm_manager.lua

```diff
@@ -697,16 +697,18 @@ function Damagelog:ReportWindow(found, deathLogs, previousReports, currentReport
 		buttons[1]:OnMousePressed()
 	end
 
-	local Logs = vgui.Create("DListView")
-	Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Time")):SetFixedWidth(40)
-	Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Type")):SetFixedWidth(40)
-	Logs:AddColumn(TTTLogTranslate(GetDMGLogLang, "Event"))
-
-	if deathLogs then
-		self:SetListViewTable(Logs, deathLogs, false)
-	end 
+	if (Damagelog.User_rights[LocalPlayer():GetUserGroup()] or 2) >= 2 then
```

Comment:
Just check here if deathLogs is truthy. You have already the check on server-side so no need to check it double.

Reply:
Then the tab wouldn't be displayed if the player has access to it, but no logs are available. Should it be like that?

Reply:
In my opinion better than an empty list.

Reply:
Can as well display `TTTLogTranslate(GetDMGLogLang, "Nothinghere")` there. I think the tab should always be visible if the player has access to view it - it will be more clear that the tab is available, just no logs were found.

Reply:
I also think the tab should be displayed since at first players wouldn't understand why the tab isn't displayed.

Reply:
This produces an error since `self` is not the DListView but ApplySchemeSettings.
`Logs:AddLine("", "", TTTLogTranslate(GetDMGLogLang, "Nothinghere"))` is working and the text in the right column.